ipatests/test_integration/test_otp.py

```diff
@@ -309,3 +311,51 @@ def test_2fa_disable_single_prompt(self):
             master.run_command(['ipa', 'user-del', USER2])
             self.master.run_command(['semanage', 'login', '-D'])
             sssd_conf_backup.restore()
+
+    def test_check_otpd_after_dirsrv_restart(self):
+        """Test for ipa-otpd when dirsrv is restart with OTP auth.
+
+        Test for : https://pagure.io/freeipa/issue/6587
+
+        When dirsrv is restarted and there is OTP authentication, then after
+        dirsrv restart there should be no failure in service ipa-otpd.
```

Comment:
Description can be simplified like:
When OTP authentication is set and dirsrv restrted, ipa-otpd service should not fail.